### ISSUE TITLE
Please switch the OmniAuth strategy

### ISSUE THREAD

**Issue by kodyclemens** [learn-co-curriculum/omniauth_lab#13]
Students that do not use Facebook (or refuse due to [growing concerns](https://en.wikipedia.org/wiki/Criticism_of_Facebook#Privacy_issues)) cannot complete this lab. Given the atmosphere of Flatiron, I believe it makes sense to use the Github OAuth.

**Comment by howardemilya** [comment]
I agree with this 100%... I skipped this lab because I refuse to have facebook.

**Comment by meg-gutshall** [comment]
You don't have to create an account. You can actually put whatever you want in the `.env` file and the tests will still pass. I used the info from the OmniAuth lesson:
```
FACEBOOK_KEY=247632982388118
FACEBOOK_SECRET=01ab234567890c123d456ef78babc901
```
